src/js/sonifier.js

```diff
@@ -0,0 +1,257 @@
+/*
+Copyright 2015-2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.defaults("floe.chartAuthoring.sonifier", {
+        gradeNames: ["floe.chartAuthoring.totalRelaying", "floe.chartAuthoring.sonifier.unitDivisorSonificationStrategy"],
+        components: {
+            textToSpeech: {
+                type: "fluid.textToSpeech",
+                options: {
+                    model: {
+                        utteranceOpts: {
+                            "lang": "en-US"
+                        }
+                    }
+                }
+
+            },
+            enviro: {
+                type: "flock.enviro"
+            },
+            synth: {
+                type: "floe.chartAuthoring.electricPianoBand",
+                options: {
+                    components: {
+                        scheduler: {
+                            type: "flock.scheduler.async",
+                            options: {
+                                synthContext: "floe.chartAuthoring.electricPianoBand"
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        model: {
+            // dataSet:,
+            // sonifiedData:
+            // sonificationQueue:
+            // currentlyPlayingData:
+            // Supplied by relaying in floe.chartAuthoring.totalRelaying grade
+            // total: {
+            //     value: number,
+            //     percentage: number
+            // }
+            isPlaying: false,
+            isTextToSpeechSupported: {
+                expander: {
+                    func: "fluid.textToSpeech.isSupported"
+                }
+            }
+        },
+        modelListeners: {
+            dataSet: {
+                funcName: "{that}.sonifyData"
+            },
+            sonificationQueue: {
+                funcName: "{that}.beginSonificationQueue"
+            }
+        },
+        // options that control playback behaviour
+        playbackOptions: {
+            // Additional gap of silence between individual data points when
+            // playing a sequence
+            gapDuration: 1,
```

Comment:
Is ```gapDuration``` in second or other unit form? Can you add a comment for it? Would be nice if there's a test for it.